src/lib/ai/mental-llama/datasets/comprehensive-test-data.ts

```
@@ -0,0 +1,703 @@
+/**
+ * MentalLLaMA Comprehensive Test Dataset
+ *
+ * This module provides an expanded set of test cases for evaluating
+ * mental health analysis prompts across different categories and scenarios.
+ */
+
+import { PromptTestCase } from '../evaluator'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)